charts/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -92,14 +92,21 @@ spec:
             - name: ETCD_TLS_SECRET_NAMESPACE
               value: {{ .Values.etcdTlsSecretNamespace | quote }}
             {{- end }}
+            {{- if .Values.openshift }}
+            # OpenShift defaults
+            - name: "ETCD_ENDPOINT_URL"
```

Comment:
These variables were already set in case ".Values.apiServerEndpointUrl" or ".Values.schedulerEndpointUrl" were not empty.

We should not set them twice and we should use the one specified by the user if both `.Values.openshift=true` and the url are set. 
Have you tested that use-case? @bpschmitt

Reply:
Can we uses something like:
```
            {{- if .Values.openshift }}
            # OpenShift defaults
            - name: "ETCD_ENDPOINT_URL"
              value: {{ .Values.etcdEndpointUrl | quote | default "https://localhost:9979" }}
            - name: "API_SERVER_ENDPOINT_URL"
              value: {{ .Values.apiServerEndpointUrl | quote | default "https://localhost:6443" }}
            - name: "SCHEDULER_ENDPOINT_URL"
              value: {{ .Values.schedulerEndpointUrl | quote | default "https://localhost:10259" }}
            - name: "CONTROLLER_MANAGER_ENDPOINT_URL"
              value: {{ .Values.controllerManagerEndpointUrl | quote | default "https://localhost:10257" }}
            {{- else }}
```